docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -34,6 +34,22 @@ appropriate for your setup:
     connection-user=exampleuser
     connection-password=examplepassword
 
+.. _clickhouse-tls:
+
+Connection security
+^^^^^^^^^^^^^^^^^^^
+
+If you have globally-trusted certificates installed on both your cluster and the
+data source, enable TLS between them by appending the ``ssl=true`` parameter
+to the ``connection-url`` configuration property:
+
```

Comment:
Here and throughout:

There is such a thing as "two-way SSL," but it's not the default and it's a PITA to set up. I would think that the coordinator and all workers would need a copy of the same client cert that is recognized and honored by the data source's server. No need for the data source to also have a cert recognized by the cluster machines for the return trip, is there? Confused.

Reply:
Good point, and I also think it needs to be said that the certificate must be configured on all Trino nodes. Updated accordingly.